### ISSUE TITLE
BLEBeacon Class

### ISSUE THREAD

**Issue by TiSiAi** [nkolban/esp32-snippets#549]
Hello
I use BLEBeacon class and I have a problem
form example:

in setMajor function, the ENDIAN_CHANGE_U16 is called
but in getMajor function the ENDIAN_CHANGE_U16 is not called !

so, if you set and get, the result is incorrect without call ENDIAN_CHANGE_U16

same problem on scan BLEDevice program:
BLEBeacon oBeacon = BLEBeacon();
 oBeacon.setData(data);
oBeacon.getMajor() > return invalid value ( not switched with ENDIAN_CHANGE_U16 )

the problem is on:
-getMajor
-getMinor
-getManufacturerId

to patch, add ENDIAN_CHANGE_U16 in each method

is it a really bug or I m wrong ?
Thank

**Comment by chegewara** [comment]
Yes, you are right. ENDIAN_CHANGE_U16  should be added to all those functions.
